Doc/library/time.rst

```diff
@@ -307,13 +312,21 @@ Functions
    .. versionchanged:: 3.10
       On macOS, the function is now system-wide.
 
+   .. versionchanged:: 3.13
+      Use the same clock as :func:`time.perf_counter`. On Windows, :func:`monotonic`
```

Comment:
Do you mean to say this only applies to Windows? Otherwise, we don’t really need two separate pairs of functions.